### DISCUSSION TITLE
[Feature]: maybe handle folder selection differently for "Nuxtr: create new page"

### DISCUSSION THREAD

**Discussion by BananaAcid** [nuxtrdev/nuxtr-vscode#104] [Ideas]
### Environment

--

### What went wrong?

1. preselect "Main pages folder" when creating new pages (as that is most common) 
2. when creating with filename "abc/info" the followup to ask to create a folder is not necessary as the folder `/pages/abc` will be created anyways and `info.vue` be placed in there

### How to reproduce it?

create a page by CMD+SHIFT+P > Nuxtr: create new page

**Comment by adhamfarrag** [comment]
Thanks @BananaAcid, I wanted to roll this out long time ago but lost attention to this! :) Thanks for mentioning it
